master/buildbot/worker/openstack.py

```diff
@@ -58,7 +58,10 @@ def __init__(self, name, password,
                  os_username,
                  os_password,
                  os_tenant_name,
+                 os_user_domain=None,
```

Comment:
This is a syntax error. Parameters with default values must come after those without defaults.